analysis/create_online_learning_curves.py

```diff
@@ -0,0 +1,94 @@
+"""Create plots for online learning."""
+
+import os
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+import pandas as pd
+from predicators.analysis.analyze_results_directory import create_dataframes
+
+pd.options.mode.chained_assignment = None  # default='warn'
+
+############################ Change below here ################################
+
+# X_KEY is the name of the metric that will be plotted on the x axis. See
+# analyze_results_directory.py for all available metrics.
+# X_LABEL is used to label the x axis.
+X_KEY = "CYCLE"  # common: CYCLE, NUM_TRANSITIONS, LEARNING_TIME
```

Comment:
is there any hope of making this script work for our offline learning experiments where we want to vary num_demos? it would be nice to have a single script since it's really the same plot... (performance vs data)

Reply:
I think that if we include NUM_DEMOs as a metric then the script should just work? If it doesn't, I will fix it once we're there.